service/handlers.go

```diff
@@ -113,3 +126,47 @@ func (e *ExecuteStruct) GetClient(projectID string) (client.RancherClient, error
 	}
 	return *apiClient, nil
 }
+
+func saveWebhook(webhookRequestData map[string]interface{}) error {
```

Comment:
passing in the entire maps makes this function have to know about too many details about what is in it. Just pass uuid and name as explicit parameters